### ISSUE TITLE
[IATI] Transform section - include conversion to spreadsheet

### ISSUE THREAD

**Issue by stevieflow** [OpenDataServices/cove#1370]
@Bjwebb do you think we might want to resituate the "Convert to spreadsheet" button that appears on XML submissions, to the new Transform section?

**Comment by Bjwebb** [comment]
Yes. And we should move the link to the converted file there from "Download and Share" also.
